examples/if_watch.rs

```diff
@@ -5,7 +5,7 @@ use ipnet::IpNet;
 #[async_std::main]
 async fn main() -> Result<()> {
     let mut watcher = IfWatcher::new().await?;
-    for _ in 0..10 {
+    for _ in 0..12 {
```

Comment:
one additional event but you're waiting for two

Reply:
the issue is the `netcat_host` example, which works on my machine, but not on ci :confused:.

Reply:
Anyway, it's two because it yields both `127.0.0.1` and `::1`.